runners/direct-java/build.gradle

```diff
@@ -147,3 +156,5 @@ createJavaExamplesArchetypeValidationTask(type: 'MobileGaming',
   gcsBucket: gcsBucket,
   bqDataset: bqDataset,
   pubsubTopic: pubsubTopic)
+
+createPortableValidatesRunnerTask(name: "validatesPortableRunner", jobServerDriver: "org.apache.beam.runners.direct.portable.job.ReferenceRunnerJobServer")
```

Comment:
Looks like there's no trailing newline. I guess `./gradlew spotlessApply` doesn't fix it because it's not a Java file.

Reply:
done